doc/guides/customizing-your-list.md

```diff
@@ -298,19 +292,23 @@ Notice the new "Actions" column at the end of our list.
 
 #### How it works
 
-- Columns with a `type` of `append` will always appear at the end of your list, even if you include them before non-`append` columns (The order in which you include `append` columns only matters relative to _each other_).
+- Columns without a `field` prop are treated as "extra columns" and will always appear at the end of your list, even if you include them before "custom columns". In the below example, the "Actions" column will appear at the end of our table, even though it's defined before the "name" column:
 
   ```tsx
-  <TodoColumn/
-    type="append"
+  <TodoColumn
+    label="Actions"
     //...Remaining props...
-  >
+  />
+  <TodoColumn
+    field="name"
+    //...Remaining props...
+  />
   ```
 
-- In our app code above, we removed the `field` prop because `append` columns represent brand new columns that inherently don't map directly to a field in our list. Despite this, we still have access to the entire row's `record` object via `renderDataCell`, so we can still access any fields from our record that we need.
-- If you want your column to appear at the beginning of your list, you can set the `type` to `prepend` instead.
+- In our app code above, we removed the `field` prop because extra columns represent brand new columns that inherently don't map directly to a field in our list. Despite this, we still have access to the entire row's `record` object via `renderDataCell`, so we can still access any fields from our record that we need.
+- If you want your column to appear at the beginning of your list, you can set the `prepend` prop to true.
 
-With all three of the `type` values we just covered--`replace`, `append`, and `prepend`--you'll notice that as we made changes to our list, Hatchify continued to render all the other columns that we _didn't_ modify as it normally would. So now let's take a look at how we can fully override Hatchify's column-rendering behavior.
+With the two types of columns we just covered - extra and custom - you'll notice that was we made changes to our list, Hatchify continued to render all the other columns that we _didn't_ modify as it normally would. So now let's take a look at how we can fully override Hatchify's column-rendering behavior.
```

Comment:
"you'll notice that was we made changes" I think this is supposed to  be "you'll notice that when we made changes"

Reply:
how did i manage to write that 🤣